daisy_workflows/image_import/windows/translate_bootstrap.ps1

```diff
@@ -87,11 +87,43 @@ function Setup-ScriptRunner {
   Run-Command reg unload 'HKLM\MountedSoftware'
 }
 
+function Test-ProductName {
+  Run-Command reg load 'HKLM\MountedSoftware' "${script:os_drive}\Windows\System32\config\SOFTWARE"
+  $pn_path = 'HKLM:\MountedSoftware\Microsoft\Windows NT\CurrentVersion'
+  $pn = (Get-ItemProperty -Path $pn_path -Name ProductName).ProductName
+  Write-Output "Product Name: ${pn}"
+  Run-Command reg unload 'HKLM\MountedSoftware'
+  $product_name = Get-MetadataValue -key 'product_name'
+  Write-Output "*${product_name}*"
+  if ($pn -like "*${product_name}*") {
+    Write-Output 'TranslateBootstrap: Product and import workflow match.'
+  }
+  else {
+    Write-Output "TranslateBootstrap: Incorrect translate workflow selected. Found: $pn, Expected: $product_name."
+  }
+}
+
+function Copy-Packages {
+  Write-Output 'TranslateBootstrap: Creating directory.'
+  $googet_dir = "${script:os_drive}\ProgramData\GooGet"
+  New-Item -Path $googet_dir -Type Directory
+  Write-Output 'TranslateBootstrap: Copying googet.'
+  Copy-Item "${script:components_dir}\googet.exe" "${script:os_drive}\ProgramData\GooGet\googet.exe" -Force -Verbose
+  Write-Output 'TranslateBootstrap: Attempt to run GooGet.'
+  & C:\ProgramData\GooGet\googet.exe -root C:\ProgramData\GooGet -help
```

Comment:
Whats the reason for this? This is running on the host system, do you mean to use "${script:os_drive}\ProgramData\GooGet\googet.exe" and $googet_dir ?

Reply:
Yeah, I wanted to verify the googet that was copied over could run on the translated system, but I see I am just running it on the host. I'll remove it.